### PR TITLE
added Dockerfile and jest-jnit as a dependency

### PR DESCRIPTION
**Pull request by dTechLife**
### What this PR does (required):
- Adds docker file that builds and runs the client/server on the same Dockerfile using express.js static routing of the build folder.

### Any information needed to test this feature (required):
- run `docker build . -t petsitter `
- run `docker run -p 80:3001 -t petsitter `

### CODE REVIEW

**Comment by dTechLife** [comment]
@navjot-chahal mind testing this? checkout the branch and run the docker commands. It will use the dot-env file you have locally. 
NOTE: Mongo db env variable needs to be set to an IP or hostname not 'localhost'.

File: server/test/auth.test.js [hunk 1]
```diff
@@ -37,7 +37,7 @@ describe("Test for all auth endpoints", () => {
         .post("/auth/register")
         .send(newUser)
         .end((err, res) => {
-          res.should.have.status(409);
+          res.should.have.status(400);
```
Comment (navjot-chahal) [inline server/test/auth.test.js]: @DestinTech Change the error code back to 409. [Here](https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/409) is the reference.

Comment (navjot-chahal) [reply server/test/auth.test.js]: I have changed that for you now!

File: server/test/auth.test.js [hunk 2]
```diff
@@ -37,7 +37,7 @@ describe("Test for all auth endpoints", () => {
         .post("/auth/register")
         .send(newUser)
         .end((err, res) => {
-          res.should.have.status(400);
+          res.should.have.status(409);
```
Comment (dTechLife) [inline server/test/auth.test.js]: very good, thank you for informing me this wasn't a typo!

**Pull request merged**